content/en/security/cloud_siem/investigator.md

```diff
@@ -11,11 +11,14 @@ further_reading:
 - link: "/cloud_siem/explorer/"
   tag: "Documentation"
   text: "Learn about the Security Signals Explorer"
+- link: "https://www.datadoghq.com/blog/visualize-cloud-activity-datadog-cloud-siem-investigator/"
+  tag: "Blog"
+  text: "Visualize activity in your cloud environment with Datadog Cloud SIEM Investigator"
 ---
 
 ## Overview
 
-<div class="alert alert-warning">Cloud SIEM Investigator currently only supports AWS CloudTrail logs.</div>
+<div class="alert alert-warning">Cloud SIEM Investigator supports AWS CloudTrail logs and Google Cloud Audit logs.</div>
```

Comment:
Should this still be a "warning"?

Reply:
Hey @christophetd! I'd would leave this "warning" because it lets users know which cloud providers we currently support. @nimishaaa what do you think?

Reply:
Makes sense, updated

Reply:
"info" is good w me! thanks!